src/sentry/interfaces/user.py

```diff
@@ -50,6 +50,9 @@ class User(Interface):
     >>> }
     """
 
+    score = 1
+    display_score = 2020
```

Comment:
This controls the order of sections in UI. In this case I tried to fix a test where the sort order was apparently unstable because two interfaces had the same score.